src/Dockerfile

```diff
@@ -6,4 +6,4 @@ RUN mkdir akka
 COPY . ./akka
 RUN ls
 WORKDIR /akka
-CMD ["/bin/bash", "-c", "x=0; while [ $x -le ${run_count} ]; do dotnet test ${test_file} ${test_filter} --framework net6.0 --logger trx;  x=$(( $x + 1 )); sleep 5s; done"]
\ No newline at end of file
+CMD ["/bin/bash", "-c", "x=1; c=0; while [ $x -le 1 ] && [ $c -le ${run_count} ]; do dotnet test ${test_file} ${test_filter} --framework net6.0 --logger trx; c=$(( $c + 1 )); if [ $? -eq 0 ]; then x=1; else x=0; fi;  done"]
```

Comment:
To prevent a situation where the test never fails! Maybe sufficient(large) `run_count` should be passed!